ebos/eclgenericvanguard.hh

```diff
@@ -84,13 +87,16 @@ public:
      * \details Needs to be in compile unit.
      */
     EclGenericVanguard();
+    EclGenericVanguard(SimulationModelParams&& params);
```

Comment:
Single-parameter constructor.  Is there a reason for not marking this `explicit`?

Reply:
no reason, fixed.